plugins/drivers/driver.go

```diff
@@ -111,6 +110,10 @@ type TaskConfig struct {
 	StderrPath      string
 }
 
+func (tc *TaskConfig) GetRawConfig() []byte {
```

Comment:
Why is this needed?

Reply:
added when debugging msgpack unmarshalling, will remove